README.md

```diff
@@ -1,6 +1,11 @@
 Shiny for Python
 ================
 
+[![Release](https://img.shields.io/github/v/release/rstudio/py-shiny)](https://img.shields.io/github/v/release/rstudio/py-shiny)
+[![Build status](https://img.shields.io/github/workflow/status/rstudio/py-shiny/Run%20tests)](https://img.shields.io/github/workflow/status/rstudio/py-shiny/Run%20tests)
```

Comment:
The badge URL has changed since this PR was created, see https://github.com/badges/shields/issues/8671

```suggestion
[![Build status](https://img.shields.io/github/actions/workflow/status/rstudio/py-shiny/pytest.yaml?branch=main)](https://img.shields.io/github/actions/workflow/status/rstudio/py-shiny/pytest.yaml?branch=main)
```